charts/portal/values.yaml

```diff
@@ -799,6 +799,9 @@ backend:
       externalRegistrationPath: "/?overlay=consent_osp"
       # -- The logic to decline an application is not yet implemented in the backend - this will currently lead to a 404 page when clicking on the link in the mail
       closeApplicationPath: "/home/decline"
+    mailingProcess:
+      # -- Client-secret for mailingprocess encryptionKey. Secret-key 'mailingprocess-encryption-key'.
```

Comment:
Is it a bit misleading to talk about a client secret here? Maybe it would be better phrase it like this: Encryption key for mailing process. ...
Same applies also for the following two:
https://github.com/eclipse-tractusx/portal-cd/blob/a9e8dc51defe0f38bdc83626179b180c194e1211/charts/portal/values.yaml#L377
https://github.com/eclipse-tractusx/portal-cd/blob/a9e8dc51defe0f38bdc83626179b180c194e1211/charts/portal/values.yaml#L795

Reply:
done :)